test/ssc_test/cmod_pvsamv1_test.cpp

```diff
@@ -145,11 +145,13 @@ TEST_F(CMPvsamv1PowerIntegration_cmod_pvsamv1, DefaultResidentialModel)
 
         ssc_number_t lcoe_nom;
         ssc_data_get_number(data, "lcoe_nom", &lcoe_nom);
-        EXPECT_NEAR(lcoe_nom, 7.03, m_error_tolerance_lo) << "Levelized COE (nominal)";
+//        EXPECT_NEAR(lcoe_nom, 7.03, m_error_tolerance_lo) << "Levelized COE (nominal)";
+        EXPECT_NEAR(lcoe_nom, -26.45, m_error_tolerance_lo) << "Levelized COE (nominal)"; // Speedy Gonzales 25yr to 2yr
```

Comment:
Negative LCOE? Might want to look at defaults for 2 year case, just to keep a meaningful value to check in the future. For 2 year runs, I set itc_fed_percent and itc_sta_percent to = [0] in pvsamv1_common_data.cpp and got a more reasonable lcoe (but a negative npv).